v6/v6_client.ml

```diff
@@ -25,21 +25,18 @@ let retry_econnrefused f =
 				None in
 		match result with
 		| Some x -> x
-		| None -> if retry then loop false else raise V6d_failure
+		| None -> if retry then loop false else raise (LicensingError V6d_failure)
 	in
 	loop true
 
-module Client = V6_interface.Client(struct
-	let rpc call =
-		retry_econnrefused (fun () ->
-			if !use_switch
-			then json_switch_rpc !queue_name call
-			else xml_http_rpc
-				~srcstr:(Xcp_client.get_user_agent ())
-				~dststr:"v6"
-				V6_interface.uri
-				call
-		)
-end)
 
+let json_url () = "file:" ^ json_path
```

Comment:
I believe a file URL needs to start with `file://`, see https://en.wikipedia.org/wiki/File_URI_scheme

Reply:
Happy to change it, but I'm curious: I've had a look at how other daemon interfaces define the json_url/json_path function, and it seems like they all follow this pattern, which would presumably result in something like this:
`file:/var/path/to/file.extension`
Given that there haven't been any issues so far (that I know of) with this format, is the change just a matter of following conventions? Also, is it worth changing for all interfaces?

Reply:
I might have raised this in the past. It's wrong but it works. So let's keep it.

Reply:
I can update this for all interfaces in a separate PR?

Reply:
Sure. Let's stick with it for the moment